BearlyHumans/squirrelville
# Camera Transitions

Added:
- Script that moves the camera along a series of points and calls events at the different points
- Script (and object in UI canvas) that fade the screen to and from black when called using a singleton pattern

Test:
- Make sure the FadeToBlack script is set up on a black image in the UI
- Add CameraTransition to a new object
- Add a reference to the camera at the top of the script
- Add points which the camera should follow as child transforms of the object
- Add freeze event to start, and unfreeze event to end (in the camera transition script inspector)
- Check play on start
- Play scene
